src/sentry/api/endpoints/organization_member/__init__.py

```diff
@@ -74,7 +74,7 @@ def can_admin_team(access: Access, team: Team) -> bool:
         return True
     if not access.has_team_membership(team):
         return False
-    return access.has_team_scope(team, "team:write")
+    return access.has_team_scope(team, "team:write") or access.has_team_scope(team, "team:admin")
```

Comment:
Ideally `has_team_scope` would take in a list of scopes to reduce the amount of calls, but that is out of scope for this PR.